final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,143 @@
+import feedparser
+import sys
+import datetime
+import argparse
+import json
+from bs4 import BeautifulSoup
+
+DEFAULT_LIMIT_COUNT = 3
```

Comment:
If user did not specify `--limit` he/she should get all available feed. fith this line I suppose user will only get 3 entries.